package.json

```diff
@@ -32,9 +35,6 @@
     "typescript": "^2.1.6"
   },
   "dependencies": {
-    "@types/bluebird": "^3.0.37",
-    "@types/lodash": "^4.14.52",
-    "@types/node": "^7.0.7",
```

Comment:
The downstream only needs these when it's building though, but if they're in dependencies they'll still be pulled in by the final "npm install" of the end user regardless of anything the downstream wants - it seems like a bit of a mess overall there